fiddler/assets/dashboards/fiddler_overview.json

```diff
@@ -0,0 +1 @@
le_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":0.8},{"comparator":">","palette":"white_on_yellow","value":0.6},{"comparator":"<","palette":"white_on_red","value":0.6}],"response_format":"scalar","queries":[{"query":"avg:fiddler.accuracy{*}","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"custom_links":[],"precision":2}},{"id":7862199840297070,"layout":{"x":37,"y":10,"width":33,"height":15},"definition":{"title":"Prediction Drift","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":0.2},{"comparator":"<","palette":"white_on_yellow","value":0.5},{"comparator":">","palette":"white_on_red","value":0.5}],"response_format":"scalar","queries":[{"query":"avg:fiddler.histogram_drift_pred_is_fraud{*}","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"custom_links":[],"precision":2}},{"id":1221242763110366,"layout":{"x":71,"y":10,"width":33,"height":15},"definition":{"title":"Traffic","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":200},{"comparator":"<","palette":"white_on_red","value":105},{"comparator":">","palette":"white_on_yellow","value":105}],"response_format":"scalar","queries":[{"query":"avg:fiddler.traffic_count{*}.rollup(count, 3600)","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"custom_links":[],"precision":2}},{"id":3293231302427438,"layout":{"x":3,"y":0,"width":23,"height":9},"definition":{"type":"image","url":"https://global-uploads.webflow.com/5e067beb4c88a64e31622d4b/623e2b19056c2266cff2d854_fiddler-ai-logo.svg","sizing":"contain","margin":"md","has_background":true,"has_border":false,"vertical_align":"center","horizontal_align":"center"}},{"id":7130745675982288,"layout":{"x":27,"y":0,"width":77,"height":9},"definition":{"type":"note","content":"Building trust into AI","background_color":"white","font_size":"56","text_align":"center","vertical_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}},{"id":4831944571744676,"layout":{"x":3,"y":94,"width":101,"height":23},"definition":{"title":"Traffic","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:fiddler.traffic_count{*}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"kcy-if8-t24"}
```

Comment:
It looks like this is using our the screenboard dashboard type. We recommend using our new dashboard as this is what will be used for integrations moving forward. Can you convert this into the snap widget dashboard?

Reply:
Changed it to the new dashboard.